src/common/DataStores/DBCStore.h

```diff
@@ -279,6 +310,12 @@ class DBCStorage
 
         T* dataTable;
         StringPoolList stringPoolList;
+
+#ifdef ELUNA
+        uint32 maxdatacount;
+        uint32 mindatacount;
+        std::unordered_map<uint32, T const*> data;
```

Comment:
I don't like how these are named. Which kind of data is this?

Reply:
its whatever data the dbc storage contains, but custom added by user.
Basically its a hack to make DBCs editable. I would like to scrap this code, but that would lose a feature. Or alternatively it could be made better as well.

Reply:
so it's in original eluna

Reply:
yes